gm4_shapeless_portals/data/gm4_shapeless_portals/advancements/light_portal.json

```diff
@@ -0,0 +1,45 @@
+{
+  "criteria": {
+    "requirement": {
+      "trigger": "minecraft:item_used_on_block",
+      "conditions": {
+        "item": {
+          "items": [
+            "minecraft:flint_and_steel",
+            "minecraft:fire_charge"
+          ]
+        },
+        "location": {
+          "block": {
+            "blocks": [
+              "minecraft:obsidian",
+              "minecraft:crying_obsidian"
+            ]
+          }
+        }
+      }
+    },
+    "module_loaded": {
```

Comment:
This is not how to properly check that the module is loaded. First of all you're using `gm4_modules` instead of `load.status` which is the patch version thing, not the actual successful load score (this will never be equal to 1 for a module downloaded from the website). Second of all, putting it in its own criterion means that if this is EVER true, then whatever players are online will have this criterion fulfilled and if it ever becomes false it doesn't matter cuz the player already fulfilled this criterion. You should check this within the `player` conditions of the `item_used_on_block` criterion. See Undead Players for an example of proper module load checking.